R/related_queries.R

```diff
@@ -1,5 +1,5 @@
 related_queries <- function(widget, comparison_item,tz,hl) {
-  i <- which(grepl("queries", widget$title) == TRUE)
+  i <- which(grepl("RELATED_QUERIES", widget$id) == TRUE)
```

Comment:
Maybe make it   `grepl("relatex_queries", widget$id, ignore.case=TRUE) == TRUE`   here and similarly in other places?

Also, is just testing for (substring) "queries" more robust?  Or is the full word better?  I really don't know.

Reply:
> Also, is just testing for (substring) "queries" more robust? Or is the full word better? I really don't know.

I do not have a strong feeling about it. Maybe full to prevent possible  extra matching.